infra/notifications/src/app.js

```diff
@@ -471,6 +472,35 @@ app.post('/send_pn', pushAppAuth, async (req, res) => {
   res.send(data)
 })
 
+const cachedCountryCodes = []
+const cachcedLangCodes = []
+app.get('/push-targets', pushAppAuth, async (req, res) => {
+  if (!cachcedLangCodes.length) {
+    const [langCodes] = await db.sequelize.query(`
+      select distinct data->>'language' as langcode 
```

Comment:
That's going to be a pretty slow query. An alternative would be to query the identity table on the 'country' column and add an index to it.